src/lmnt/api.py

```diff
@@ -121,7 +121,7 @@ async def synthesize(self, text, voice, **kwargs):
     form_data = aiohttp.FormData()
     form_data.add_field('text', text)
     form_data.add_field('voice', voice)
-    form_data.add_field('seed', kwargs.get('seed', 0))
+    form_data.add_field('seed', kwargs.get('seed'))
```

Comment:
should we omit entirely?

e.g.

if kwargs.get('seed') is not None:
  form_data.add_field(...)